docs/api/_data/package-registration-index.json

```diff
@@ -2,11 +2,11 @@
   "count": 1,
   "items": [
     {
-      "@id": "https://api.nuget.org/v3/registration3/nuget.server.core/index.json#page/3.0.0-beta/3.0.0-beta",
+      "@id": "https://api.nuget.org/v3/registration-sample/nuget.server.core/index.json#page/3.0.0-beta/3.0.0-beta",
```

Comment:
Is there any example or guidance on how to get this `registation-sample` value from index page? Otherwise, takes some time to figure out what it's.

Reply:
It's in the service index page, and at the top of the document.

Reply:
Do you mean this part? I don't see anything particular about selecting correct registration url.

![image](https://user-images.githubusercontent.com/8766776/208972760-629dcafe-cabf-4d27-b2b8-4458b749454d.png)

Reply:
![image](https://user-images.githubusercontent.com/94054/208973272-8b194d64-ddcf-4208-92c0-f72238aa9055.png)

Reply:
Here we have 2 base urls. Currently they're same, but in case they're different then which one to use?
https://user-images.githubusercontent.com/8766776/207996385-d22133dd-79c4-455d-a9b2-b55703ff8c56.png

Reply:
The `/Versioned` one is undocumented (in line with https://learn.microsoft.com/en-us/nuget/api/overview#undocumented-resources-on-nugetorg) and should not be used. You select the `@type` you want to use based on https://learn.microsoft.com/en-us/nuget/api/registration-base-url-resource#versioning.